components/log-viewer-webui/server/package.json

```
@@ -14,6 +14,8 @@
   "license": "Apache-2.0",
   "type": "module",
   "dependencies": {
+    "@aws-sdk/client-s3": "^3.726.1",
+    "@aws-sdk/s3-request-presigner": "^3.726.1",
```

a new version was just released. let's run `npm i` after updating this, so that the package-lock file can also be updated.
```suggestion
    "@aws-sdk/client-s3": "^3.731.0",
    "@aws-sdk/s3-request-presigner": "^3.731.0",
```